### ISSUE TITLE
Add accessibility remediation request form and button

### ISSUE THREAD

**Issue by carakey** [osulp/Scholars-Archive#2694]
### Descriptive summary

This ticket is part of the larger ADA Title II compliance effort. 

Users should be able to request an accessible version of a work, with as little burden to them as possible. This ticket is to add an accommodation request pathway: "Request Accessible Version" button on work pages, which leads to a pre-populated form, which triggers an email to SA admins for remediation and follow up. 

See the implementation of this from UNC in the Carolina Digital Repository: [example work](https://cdr.lib.unc.edu/concern/masters_papers/ft848v59q?locale=en) + [request form](https://blogs.lib.unc.edu/cdr/accessibility-request/). 

I can follow up with specific language to use.

**Comment by lamtu1** [comment]
Hi @carakey, I wanted to follow up with you on this:

I would like to know how you want the language is use in the `accessibility request form` to have when the user click the button on the work?

What type of `field(s)` would the page have and which information needed to be `pre-populated`? And also I was wondering if the button placement below is good when ask for the request? Thank you!

![Image](https://github.com/user-attachments/assets/ef0ab619-7026-4537-9204-5586524cac2f)

**Comment by carakey** [comment]
Hi @lamtu1 --

1. The button placement looks good to me.
2. Here is a mockup of the form with fields and text (at least preliminary; approved by @mellinma) https://docs.google.com/document/d/1ClbH1C8Mk7RRCr1-crK1RYhMMMXMxbch3GEBgMIooyM 
3. If possible we would like to have the form built/hosted in Drupal as part of the library website so that we could extend its use to other systems beyond SA. 
    - To make that work, we would need to do email notification routing based on the system, so wondering if that works at all and if it can be derived from the `Link` field or if we would need a separate field, such as selecting the system from a dropdown. (Probably getting ahead of myself.) 
    - You asked about pre-populating fields. It would be great to have the referring URL populate the `Link` field, but it seems like that may get complicated in Drupal / if used for multiple systems.

**Comment by carakey** [comment]
I don't see the button to access the request form on any works on Staging.

**Comment by simholt** [comment]
Good stuff: Button shows up on existing and new works. I submitted a request -- it autofilled my information and the request showed up in the SA email.
Question: I was not able to see if users who were not logged in can make requests. When logged out, I could not access show pages; it just gave the broken message.

**Comment by lamtu1** [comment]
@simholt, it is like maybe a routing thing that I may need to implement a back button to guide the user back to the show page or either redirect them back to show page once submission is done. It is something for me to dig into

**Comment by simholt** [comment]
@lamtu1 I don't think so -- something similar happened on prod last week. Let @CGillen have a look first!

**Comment by lamtu1** [comment]
@simholt Got the update that Corey fix in staging, can you tested out again when you get the chance? Thank you!

**Comment by simholt** [comment]
It worked!
QA pass!

**Comment by simholt** [comment]
@lamtu1  This is not working on production. An "Undeliverable: Scholars Archive Accessibility Request Form: Request Confirmation" email showed up this weekend because the requestor had put their email in wrong. I did a test request and was sent the "Request Confirmation" email as a user, but nothing about the request came to the SA email account. Looks like something is awry!

**Comment by lamtu1** [comment]
@simholt I will take a look into it, that does seem weird

**Comment by lamtu1** [comment]
`Note:` Add in a confirm email field to double check that both email enter is correct.

**Comment by simholt** [comment]
QA pass -- it blocked the message when the emails didn't match.
